nixtla/nixtla_client.py

```diff
@@ -654,13 +651,13 @@ def _get_model_params(self, model: str, freq: str) -> Tuple[int, int]:
 
     def _maybe_assign_weights(
         self,
-        weights: Optional[Union[list[float, list[list[float]]]]],
+        weights: Optional[Union[List[float, List[List[float]]]]],
```

Comment:
Sorry, I think this was already wrong.
```suggestion
        weights: Optional[Union[List[float], List[List[float]]]],
```